src/pages/tug/apply/Apply.component.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react';
+import Router from 'next/router';
+import { Modal, Button } from 'antd';
+
+import styles from './Apply.module.scss';
+import Container from '~/components/Container/Container';
+import { CoreLayout } from '~/layouts';
+import Form from './form';
+import { api } from '@tidb-community/datasource';
+
+const SuccessModal = (props) => (
+  <Modal title="提交成功！" {...props}>
+    <p>
+      我们将于 3 个工作日内对您提交的材料进行审核，并以【AskTUG 私信】形式告知您结果。
+      若审核通过，【微信小助手】将添加您为好友并邀请您进入 TUG 专属交流群，敬请留意。
+    </p>
+    <p>【AskTUG 私信】位置：社区导航栏 - 问答论坛 - 右上角头像下拉 - 私信</p>
+  </Modal>
+);
+
+const Apply = () => {
+  const title = 'TiDB User Group 会员申请';
+
+  const postImageUrl = '/images/people/apply/post.png';
+
+  const question = 'TUG 是什么';
+  const answer = 'TUG 是汇聚全球数据库、⼤数据技术从业者的社区，是⼀个独⽴、⾃发、不以盈利为⽬的的组织。';
+
+  const whyJoinTitle = '加⼊TUG ，你能获得什么？';
+  const whyJoinItems = [
+    { index: 1, iconUrl: '/images/people/apply/why-join-1.svg', text: '让数据库、⼤数据从业者找到⾃⼰的圈⼦' },
+    { index: 2, iconUrl: '/images/people/apply/why-join-2.svg', text: '探索技术问题，随时随地交流成⻓，解决问题' },
+    { index: 3, iconUrl: '/images/people/apply/why-join-3.svg', text: '发表技术⻅解，收获前沿知识，提升个⼈影响⼒' },
+  ];
+
+  const [successModal, showSuccessModal] = useState(false);
+  const [failModal, showFailModal] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [fieldMessage, setFieldMessage] = useState(undefined);
+
+  const FailModal = (props) => (
+    <Modal title="提交失败" {...props}>
+      <p>{errorMessage ?? 'System Error'}</p>
+      {!!fieldMessage && (
+        <ul>
+          {Object.keys(fieldMessage).map((key) => (
+            <li key={key}>
+              {key}: {fieldMessage[key].join(', ')}
+            </li>
+          ))}
+        </ul>
+      )}
+    </Modal>
+  );
+
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const handleSubmit = (data) => {
```

Comment:
The was a helper already handled field errors https://github.com/pingcap/tug-website/blob/41aa50f92b1619cf115007efcfb92efb54ad80a4/packages/common/src/utils/form/index.js#L70